devops/docker/Dockerfile.linting

```diff
@@ -15,4 +16,6 @@ RUN sudo pip install pyenchant==2.0.0
 WORKDIR /src
 COPY . /src
 
-CMD ["sudo", "make", "--keep-going", "lint"]
+RUN sudo make install-mypy
```

Comment:
For caching reasons, this should be above the `COPY` command. Also, shouldn't we mount `$PWD:/src` instead of copying everything in to the container? It makes for a marginally faster edit/test cycle.

Reply:
this docker image is designed to work for CI that does not allow mounting volumes and contrary to the admin and securedrop images, it does not have the required special case that would allow mounting . We could have a separate PR to optimize this for development and allow mounting. If you feel strongly this must be done for this PR to be merged I'd be happy to do the needful.